.travis.yml

```diff
@@ -1,8 +1,17 @@
 language: python
 python:
     - "2.7"
+before_script:
+  - git config --global user.email "travis"
+  - git config --global user.name "Travis CI"
+  - git remote rm origin
+  - git remote add origin https://user:${travis_deploy_key}@github.com/fossasia/fossasia.org.git
```

Comment:
The private key is not a password.
I do this using the private key: 

    ssh-agent bash -c "ssh-add .ssh/id_rsa; git push"